test/RotateLabels.cxx

```diff
@@ -102,6 +102,26 @@ RotateNTimes(typename ImageType::Pointer input,
 
 // The BSpline interpolator has more arguments than other interpolators, so we set the additional parameter to the default value
 template<class TImage,typename TCoordRep> class BSplineInterpolator : public itk::BSplineInterpolateImageFunction<TImage,TCoordRep> {};
+template<class TImage,typename TCoordRep> class BSplineInterpolator0 : public itk::BSplineInterpolateImageFunction<TImage,TCoordRep> {
+  public:
+  /** Standard class type aliases. */
+  using Self = BSplineInterpolator0;
+  using Superclass =
+    itk::BSplineInterpolateImageFunction<TImage,TCoordRep>;
+  using Pointer = itk::SmartPointer<Self>;
+  using ConstPointer = itk::SmartPointer<const Self>;
+
+  /** Method for creation through the object factory. */
+  itkNewMacro(Self);
+
+  /** Run-time type information (and related methods). */
+  itkTypeMacro(Self, Superclass);
+
+  BSplineInterpolator0() {
+    this->SetSplineOrder( 0 );
```

Comment:
The test timed out in another PR which only made style changes. I suggest to restore the original order, and merge.